Add section about linking JS stack trace to the source code (#10566)
DataDog/documentation: content/en/real_user_monitoring/error_tracking/browser_errors.md

  text: "Error tracking"
---

Error Tracking processes errors collected from the browser by the RUM SDK: whenever a [source][1] or [custom][2] error containing a stack trace is collected, Error Tracking processes and groups it under an issue (group of similar errors). To quickly get started with error tracking:

1. Download the latest version of the [RUM Browser SDK][3].
2. Configure the __version__, the __env__ and the __service__ when [initializing your SDK][4].

> 3. Upload JS source maps [following our guide][5] to get unminified stack traces.

Reviewer: ```suggestion
3. Upload JavaScript source maps [following our guide][5] to get unminified stack traces.
```

## Link errors with your source code

> In addition to sending source maps, the new version of the [Datadog CLI][6] reports git information such as the commit hash, the repository url as well as the list of tracked file paths in the git repository. Using this information, Error Tracking and RUM will make it even easier to correlate an error with the source code by allowing you to pivot from any stack trace frame to the related source code in your source code management tool. 

Reviewer: ```suggestion
In addition to sending source maps, the new version of the [Datadog CLI][6] reports Git information such as the commit hash, the repository URL, and the list of tracked file paths in the Git repository. Using this information, Error Tracking and RUM can correlate an error with the source code, and you can pivot from a stack trace frame to the related source code in your source code management tool. 
```

{{< img src="real_user_monitoring/error_tracking/link_to_git_js_example.gif" alt="Link from a stack frame to the source code"  >}}

> <div class="alert alert-info">To see links to the code, make sure to use the <code>0.12.0</code> version or more of the <a href="https://github.com/DataDog/datadog-ci/tree/master/src/commands/sourcemaps#sourcemaps-command">Datadog CLI</a>.</div>

Reviewer: ```suggestion
<div class="alert alert-info">Linking from stack frames to source code is supported in the <a href="https://github.com/DataDog/datadog-ci/tree/master/src/commands/sourcemaps#sourcemaps-command">Datadog CLI</a> version <code>0.12.0</code> version and later.</div>
```
